### DISCUSSION TITLE
Copilot Subscription Payment Issue

### DISCUSSION THREAD

**Discussion by devZaidGit** [community/community#66097] [Copilot Conversations] [answered]
### Select Topic Area

Copilot Subscription Payment Issue

### Body

I encountered a billing issue when trying to purchase a Copilot subscription. The error message is:

"Cannot proceed with payment
There appears to be an issue with the billing information. We're looking into it and will send you an update within 48 hours. Thank you for your patience."

Please assist in resolving this so I can access Copilot.

**Comment by arafajs** [comment]
1. Check your credit card information to make sure it is correct.
2. Try using a different credit card or payment method.
3. Contact your credit card company to see if there are any restrictions on international payments.

- Go to the GitHub Support page.
- Click on the "Contact Support" button.
- Select the "GitHub Copilot" product.
- Select the "Billing" issue type.
- Enter your contact information and a brief description of the issue.
- Click on the "Submit" button.
- GitHub support will usually respond to your request within 48 hours.

**Reply by devZaidGit** [reply]
I encounter an error before reaching the payment page, and therefore, I don't proceed to the payment method. @arafajs

**Comment by Akash1134** [comment, marked as answer]
Hi @devZaidGit,  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them 🙂

**Reply by queenslaynaila** [reply]
Hey can i get help @Akash1134 i succesfully completed the payment confirmed it with my payment provider it was succeful, but copilot wont just recongise the payment, it still says subscription expired, this partciular payment is not even in the payment history, i have opened a ticket reagrding this for a while no response, kindly assist, ticket has been opened no assistant , ticket no #3121302

**Reply by andrewhaller** [reply]
Link takes you to a dashboard. This answer is useless and complete garbage.

**Reply by Kamil777** [reply]
Uploading 98675016b3c84bbaaf5ff69261213ef0.mp4…

**Comment by tadeotech** [comment]
Did they get back to you? I just had the same issue, and I know that the place to raise support tickets is the support page, I guess the reason we also turn here is to see if we can get any answers or explanations quickly. The ugly part of support is when you don't even get a receipt of your ticket, and when there's no clarity as to what we should expect from the service. I did open a ticket, but who knows when they're gonna get back to me, could be the actual 48h, could be never. Has anyone solved this somehow?

**Reply by RunsFor** [reply]
Same thing, and no auto-response in email

**Reply by gothicsoftware** [reply]
I got an auto-response email. But it has been 24+ hours and no contact. I did not get to the step to enter my credit card. They don't have it. I entered my name, then address, then hit next.. I released I wanted my home address and hit previous..I entered my home address and when I hit next, I got the message. I cannot do anything else. I also cannot build and deploy and I can not even pay them. This is nuts.

**Reply by mrbombadil** [reply]
same here

**Comment by CarlosUlisesOchoa** [comment]
Same problem 🙁

![image](https://github.com/community/community/assets/26280134/61903d7a-1dc9-43dd-91e0-f6b5633ce7dd)

Update: After few hours I received this email from GitHub team and as they said. My issue was fixed, thanks a lot !

![image](https://github.com/community/community/assets/26280134/ba214f4b-7b60-4e2d-a330-e8afd94f4267)

**Comment by subhash-meter** [comment]
Here are the workarounds.
Workaround1:
If your address is 100 Clearview Dr
Can you please try entering the 100 Clearview Dr into the first address line, with #100 in address line 2? Let me know if you still see an error. 
 
Workaround2:
One potential workaround that had success with is using the address's 9-digit zip code. If you are unsure of this zip code, you can look it up using this USPS tool: 
 
https://tools.usps.com/zip-code-lookup.htm?byaddress

**Comment by uriinf** [comment]
Try to disable browser extensions which blocks third-party trackers. I have Privacy Badger extension, which was blocking `*.stripe.com` on the Payment method update page. Once I've disabled the extension - card is added without any issues.

**Comment by adnansh4u** [comment]
I'm having this issue:
Your account is currently locked from purchases. Please update your payment information.

**Comment by Haytham2210** [comment]
You can't proceed with your payment
There appears to be an issue with the billing information. We're looking into it and will send you an update within 48 hours. Thank you for your patience.

**Comment by ashwina123** [comment]
I am not able to renew my copiolet subscription. i have entered payment details but nothing happening

**Comment by AniketMalik** [comment]
Hey Guys, i am from India and have not been able to pay for the Copilot Sub. for about a month now. The card that i used to pay ( which worked previously ) is in perfect condition yet with github it doesn't work. I have tried 4-5 different banks cards from visa to mastercard yet nothing works now.

There's no option in the iOS application for paying as well.

Kindly share any tips on how i can clear the dues???

**Comment by Samuka-Vey** [comment]
Hi everyone,

I recently had my GitHub Copilot for Students plan active, and I noticed that after I added my credit card and tried enabling Copilot Pro, my billing page still says:

“You are eligible to use GitHub Copilot for free — $0.00”
However, inside VS Code I now have access to Copilot Chat, code explanations, and all the Copilot Pro features — even though my account still shows the student license.

I just want to confirm a few things:

Am I currently using the same features as Copilot Pro (just free under the Student license)?

Do I need to cancel anything now to avoid automatic billing later?

**Comment by darclins** [comment]
How to change my payment information if it has been blocked?
<img width="1227" height="324" alt="image" src="https://github.com/user-attachments/assets/e8886e96-5396-4cd6-87a0-7698b5c17114" />

**Comment by sumitkdevcode** [comment]
I have 2 year Github student pack but its showing fir payment can solve this issue.![image](https://github.com/user-attachments/assets/6155cc9a-d525-4711-a7ee-e3d251a3eab0)

**Reply by solomon-211** [reply]
This is the same problem I am facing right now, what can be the problem

**Reply by sumitkdevcode** [reply]
Add any international payment card

**Comment by roshanjaiswal** [comment]
<img width="1577" height="922" alt="image" src="https://github.com/user-attachments/assets/ca0cc89d-a2a5-4a6e-abcb-f7576abf9df0" />

**Comment by elkhatmx2025** [comment]
hello , i paid but sill telling me that gent can only be used for paid option

**Reply by elkhatmx2025** [reply]
I have paid 48 USD and it's not working on m visual studio , keep saying there is no agent , what the hell

**Reply by elkhatmx2025** [reply]
please look into this i have made the payment but is not working on my stujdio

**Comment by alexeyalexandrov2026-tech** [comment]
wtf !?!?!?why i cant upgrade???and no body answer  me from suport
